sdk/storage/test/datalake/file_system_client_test.cpp

```diff
@@ -0,0 +1,238 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "file_system_client_test.hpp"
+
+#include <algorithm>
+
+namespace Azure { namespace Storage { namespace Test {
+
+  const size_t c_PATH_TEST_SIZE = 5;
+
+  std::shared_ptr<DataLake::FileSystemClient> FileSystemClientTest::m_fileSystemClient;
+  std::string FileSystemClientTest::m_fileSystemName;
+
+  std::vector<std::string> FileSystemClientTest::m_PathNameSetA;
+  std::vector<std::string> FileSystemClientTest::m_PathNameSetB;
+  std::string FileSystemClientTest::m_DirectoryA;
+  std::string FileSystemClientTest::m_DirectoryB;
+
+  void FileSystemClientTest::SetUpTestSuite()
+  {
+    m_fileSystemName = LowercaseRandomString(10);
+    m_fileSystemClient = std::make_shared<DataLake::FileSystemClient>(
+        DataLake::FileSystemClient::CreateFromConnectionString(
+            ADLSGen2ConnectionString(), m_fileSystemName));
+    m_fileSystemClient->Create();
+
+    m_DirectoryA = LowercaseRandomString(10);
+    m_DirectoryB = LowercaseRandomString(10);
+    for (unsigned i = 0; i < c_PATH_TEST_SIZE; ++i)
```

Comment:
`size_t` or `std::size_t`? because `c_PATH_TEST_SIZE` is `size_t`

Reply:
`size_t`, more concise.